durable-storage/src/persistence_layer.rs

```diff
@@ -289,30 +289,23 @@ impl Drop for PersistenceLayer {
 }
 
 #[cfg(test)]
-mod tests {
+pub(crate) mod utils {
     use std::path::Path;
-    use std::path::PathBuf;
 
-    use proptest::prelude::Strategy;
-    use proptest::prelude::any;
-    use proptest::proptest;
-    use rocksdb::properties::ESTIMATE_NUM_KEYS;
     use tempfile::TempDir;
 
-    use super::*;
-
-    struct TestableTmpdir {
+    pub(crate) struct TestableTmpdir {
```

Comment:
I think you don't need `pub(crate)` here again - because the module is already `pub(crate)`? 

But feel free to resolve :)